addons/mrp_subcontracting/models/stock_warehouse.py

```diff
@@ -166,16 +184,32 @@ def _get_picking_type_update_values(self):
                 'default_location_src_id': production_location_id.id,
                 'default_location_dest_id': subcontract_location_id.id,
                 'barcode': self.code.replace(" ", "").upper() + "-RESUPPLY",
+                'active': self.subcontracting_to_resupply
             },
         })
         return data
 
-    def _get_subcontracting_location(self):
+    def _get_reference_subcontract_location(self):
```

Comment:
yes, because there can be more than one subcontracting location, the reference (created automatically and related to the company) and any other location marked as subcontracting by the user. so the old name does not specify what location it returned

Reply:
But in theory it doesn't make sense for users to mark subcontracting locations that aren't sub-locations of the company subcontracting location. It's true there is flexibility so users can do it if they really want to, but then the name `_get_reference_subcontract_location` will be just as confusing.